### ISSUE TITLE
Rust support

### ISSUE THREAD

**Issue by zeebo** [storj/drpc#15]
Tracking issue for Rust bindings.

I have some started at https://github.com/zeebo/drpc-rs but I need help integrating into the Rust ecosystem.

**Comment by gembin** [comment]
No progress for a long time...

**Comment by zeebo** [comment]
Indeed. I have very little rust experience (this is my first project!). I just pushed some code up that attempts to make it work with async, but I don't know if that's right. There's a bunch of questions I have about the rust ecosystem. For example:

- Which async driver library should I use? Is it possible to be agnostic to it?
- Is async_trait the right thing? Seems hacky!
